k8s/local/aztec-listener/sandbox/deployment.yaml

```
@@ -49,6 +49,6 @@ spec:
               value: "true"
             - name: AZTEC_LISTEN_FOR_PENDING_TXS
               value: "true"
-            - name: AZTEC_RPC_URL
-              value: "http://aztec-sandbox-node:8081"
+            - name: AZTEC_RPC_URL_POOL
```

```suggestion
            - name: AZTEC_RPC_URLS
```

(The const does not need to say how the implementation works.)

---

changed name